### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [x] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [x] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [x] Renaming `**model` to `**table**` (only for transform repos)
    - [ ] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -61,6 +67,19 @@ body:
         - other (mention it in "Additional Context")
     validations:
       required: true
+  - type: dropdown
+    id: orchestration_type
+    attributes:
+      label: How are you running this dbt package?
+      multiple: true
+      options:
+        - Quickstart
```
Comment (fivetran-joemarkiewicz) [inline .github/ISSUE_TEMPLATE/bug-report.yml:76]:
```suggestion
        - Fivetran Quickstart Data Model
```

File: README.md [hunk 2]
```diff
@@ -11,15 +11,15 @@
...
 - Currently supports the following Fivetran app platform connectors:
     - [Apple App Store](https://github.com/fivetran/dbt_apple_store)
     - [Google Play](https://github.com/fivetran/dbt_google_play)
 - Generates a comprehensive data dictionary of your source and modeled App Reporting data via the [dbt docs site](https://fivetran.github.io/dbt_app_reporting/)
 
 <!--section="app_reporting_transformation_model"-->
-Refer to the table below for a detailed view of final models materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_app_reporting/#!/overview) for more details about these models. 
+Refer to the table below for a detailed view of final models materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_app_reporting/#!/overview) for more details about these models.
 
 | **model**                  | **description**                                                                                                                                               |
```
Comment (fivetran-joemarkiewicz) [inline README.md:24]:
```suggestion
| **Table**                  | **Description**                                                                                                                                               |
```

File: README.md [hunk 3]
```diff
@@ -11,17 +11,17 @@
...
+## What does this dbt package do?
+- Standardizes schemas from various app platform connectors and creates reporting models for all activity aggregated to the device, country, OS version, app version, traffic source and subscription levels.
 - Currently supports the following Fivetran app platform connectors:
     - [Apple App Store](https://github.com/fivetran/dbt_apple_store)
     - [Google Play](https://github.com/fivetran/dbt_google_play)
 - Generates a comprehensive data dictionary of your source and modeled App Reporting data via the [dbt docs site](https://fivetran.github.io/dbt_app_reporting/)
 
 <!--section="app_reporting_transformation_model"-->
-Refer to the table below for a detailed view of final models materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_app_reporting/#!/overview) for more details about these models. 
+Refer to the table below for a detailed view of final models materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_app_reporting/#!/overview) for more details about these models.
```
Comment (fivetran-joemarkiewicz) [inline README.md:22]:
```suggestion
Refer to the table below for a detailed view of final tables materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_app_reporting/#!/overview) for more details about these tables.
```

**Review by fivetran-avinash — APPROVED** [review]
After a few tweaks, this is now good to go.

**Pull request merged**